freqtrade/loggers.py

```diff
@@ -0,0 +1,50 @@
+import logging
```

Comment:
I like splitting out logic to files, so it's easy to open them.

This got me thinking if we should have a module (folder) for these things to keep the structure cleaner... (think `freqtrade.utils.loggers`).

to be clear, It's just a thought asking for opinion - not something i'd want to do in this PR ...